### PR TITLE
coq-matching-logic

### PR DESCRIPTION
**Pull request by h0nzZik**

### CODE REVIEW

File: extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+license: "LGPL"
+
+synopsis: "Matching Logic in Coq"
+description: """
+A library for Matching Logic."""
+
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "coq" {>= "8.12" & < "8.13~"}
+]
```
Comment (palmskog) [inline extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam:17]:
```suggestion
  "dune" {>= "2.5"}
]
```
If you want to use Dune, you need to include it as a dependency.

File: extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam [hunk 2]
```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "jenda.tusil@gmail.com"
+version: "dev"
```
Comment (palmskog) [inline extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam:3]:
```suggestion
```
This line is redundant.

File: extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+homepage: "https://github.com/harp-project/AML-Formalization"
+dev-repo: "git+https://github.com/harp-project/AML-Formalization.git"
+bug-reports: "https://github.com/harp-project/AML-Formalization/issues"
+license: "LGPL"
```
Comment (palmskog) [inline extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam:8]: Which version of LGPL is this? Please use an identifier from here: https://spdx.org/licenses/

**Review by palmskog — CHANGES_REQUESTED** [review]

File: extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "category:Mathematics/Logic"
+]
+authors: [
+  "Jan Tusil"
+  "Lucas Peña"
+  "Attila Boros"
+  "Mizsei Tamás"
+]
+url {
+  src: git+https://github.com/harp-project/AML-Formalization.git#master
```
Comment (palmskog) [inline extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam]:
```suggestion
  src: "git+https://github.com/harp-project/AML-Formalization.git#master"
```
This must be quoted to be valid. Please run `opam lint opam` locally before submitting a new version.

**Comment by h0nzZik** [comment]
@palmskog I addressed your comments but made multiple commits/pushes. Can you kill CI builds for the older commits?

**Review by palmskog — APPROVED** [review]

**Pull request merged**